imagenameparser/api/image_name_regexp.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2021 SUSE LLC
```

Comment:
This duplicates the same module in mashhouse which is likely to change. Might be worth considering a way to share code instead of duplication. Granted the shared code would have to be in this module given mashhouse is private.

Reply:
That is the plan. Once this repository is merged, we will cleanup the mashhouse code to import module from this repo.